_includes/abstracts/2024/inspecta.html

```diff
@@ -0,0 +1,32 @@
+<!--<title>Summit abstract</title>-->
+<!--
+    Copyright 2024, seL4 summit authors (see abstracts)
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+<h4 class="summit-abstract-title">
+    Industrial Scale Proof Engineering for Critical Trustworthy Applications (INSPECTA)
+</h4>
+<p class="summit-abstract-type">
+    Keynote
+</p>
+<p class="summit-abstract-author">
+    Presented by Darren Cofer, <span class="summit-abstract-affiliation">Collins Aerospace</span>
+</p>
```

Comment:
See my comments below. Here again, you could have links from Darren's name to his page/linkedIn and from Collins to the company's website (at least for the keynotes, but could be for all the ones we have later)